src/ast/ast.h

```diff
@@ -632,9 +632,9 @@ class AttachPoint : public Node {
   uint64_t func_offset = 0;
   bool ignore_invalid = false;
 
-  std::string name(const std::string &attach_point) const;
-  std::string name(const std::string &attach_target,
-                   const std::string &attach_point) const;
+  std::string name() const;
+
+  AttachPoint match_override(const std::string &match) const;
```

Comment:
#bikeshedding: the name `match_override` sounds like it returns a bool. Maybe `create_ap_match` or just `create_match`

Reply:
Good point, it seems that using `match` is a bit confusing in this context. What about `expansion_copy` or `create_expansion`? Personally, I like the former as it clearly states that a copy is created.

Reply:
Went with `create_expansion_copy` in the end. It's a bit verbose but expresses the intent the best.